src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -1,47 +1,126 @@
 import { AboutArtistFragmentContainer as AboutArtist } from "./Components/AboutArtist"
-import { AboutWorkFragmentContainer as AboutWork } from "./Components/AboutWork"
-import { ArtworkActionsFragmentContainer as ArtworkActions } from "./Components/ArtworkActions"
-import { ArtworkAvailabilityFragmentContainer as ArtworkAvailability } from "./Components/ArtworkAvailability"
+import { AboutWork } from "./Components/AboutWork"
+import { ArtworkAvailability } from "./Components/ArtworkAvailability"
 import { ArtworkDetailsFragmentContainer as ArtworkDetails } from "./Components/ArtworkDetails"
+import { ArtworkHeader } from "./Components/ArtworkHeader"
 import { ArtworkTombstoneFragmentContainer as ArtworkTombstone } from "./Components/ArtworkTombstone"
-import { ImageCarouselFragmentContainer as ImageCarousel } from "./Components/ImageCarousel/ImageCarousel"
 import { OtherWorksFragmentContainer as OtherWorks } from "./Components/OtherWorks"
-import { SellerInfoFragmentContainer as SellerInfo } from "./Components/SellerInfo"
+import { SellerInfo } from "./Components/SellerInfo"
 
 interface Props {
   artwork: Artwork_artwork
 }
 
+interface State {
+  sections: Array<{
+    type: "tombstone" | "availability" | "sellerInfo" | "aboutWork" | "details" | "aboutArtist" | "otherWorks"
+    data: any
+  }>
+}
+
 export class Artwork extends React.Component<Props> {
-  render() {
+  state: State = {
+    sections: [],
+  }
+
+  componentDidMount() {
+    const { artwork } = this.props
+    const sections = []
+
+    sections.push({
+      type: "header",
+    })
+
+    sections.push({
+      type: "tombstone",
+    })
+
+    if (artwork.availability) {
+      sections.push({
+        type: "availability",
+        data: artwork.availability,
+      })
+    }
+
+    if (artwork.partner && artwork.partner.name) {
+      sections.push({
+        type: "sellerInfo",
+        data: artwork.partner.name,
+      })
+    }
+
+    if (artwork.description || artwork.additional_information) {
+      sections.push({
+        type: "aboutWork",
+        data: { description: artwork.description, additionalInformation: artwork.additional_information },
+      })
+    }
+
+    sections.push({
+      type: "details",
+    })
+
+    sections.push({
+      type: "aboutArtist",
+    })
+
+    sections.push({
+      type: "otherWorks",
+    })
+
+    this.setState({ sections })
+  }
+
+  renderItem = ({ item: { data, type } }) => {
     const { artwork } = this.props
+    switch (type) {
+      case "header":
+        return <ArtworkHeader artwork={artwork} />
+      case "tombstone":
+        return <ArtworkTombstone artwork={artwork} />
+      case "availability":
+        return <ArtworkAvailability availability={data} />
+      case "sellerInfo":
+        return <SellerInfo name={data} />
+      case "aboutWork":
+        return <AboutWork additionalInformation={data.additionalInformation} description={data.description} />
```

Comment:
We talked about this over slack but putting here for posterity. 😄

For components that _are_ tied to the data they fetch, I'm not sure I see the need for passing in props here vs. leaving it as a fragment container. In general it seems best to utilize relay's tools, and in this case we're just adding overhead-- if someone were to delete the additional information component, they'd have to know to delete these specific fields from the query vs. simply removing the fragment.

I'll default to @alloy or @l2succes in case there are things I'm missing tho!

Reply:
I 100% agree with @sweir27.

Additionally this overhead is worse because `data` seems to be untyped, which of course can be fixed, but is one more piece of overhead.

Reply:
I’m interested to learn why this change was made, though?

Reply:
My understanding of the `FlatList` component is that it will create a render pass on `renderItem` for each index of the data array. Therefore we wouldn't be able to use `ItemSeparatorComponent` or similar props if the data array isn't the exact length of the sections we want to render. So I'm not sure why we would want to have a static data array. My thought was that we would want to pull the data out of the `Artwork` relay query that is relevant to which sections we want to render so the sections are the correct length. Would love to pair to clear this up.

Reply:
I'll defer to @sweir27 for context on edge cases of the Artwork pages. My only suggestion is: if it's possible for the artwork view to a variable number of sections. For instance, in the case that there are no other works, @kierangillen's current implementation would be the most flexible, namely pushing sections onto an array. We found that to work best for the Show and Fair views.

But if you can count on always having the same number of sections and section types for all artworks, as @alloy and @sweir27 mentioned, using a static array and leaving all section wrapped in fragment containers is preferable.

Reply:
Sections should be determined in the component that renders the list, as this implementation currently does and which I acknowledged [here](https://github.com/artsy/emission/pull/1674#discussion_r295334250), but that’s orthogonal to using fragment containers or not. Perhaps I’m missing something in your reasoning @l2succes?

Reply:
No I think we're on the same page! Sorry for the confusion, I was just answering @kierangillen's point about how to create the sections array based on relay data. 

re: fragment containers. I agree with @sweir27 as well